README.md

```diff
@@ -1 +1,89 @@
-# capstone_one
\ No newline at end of file
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+
+- [LIVE]
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [CapstoneProject] <a name="about-project"></a>
+[CapstoneProject] is a capstone project given by Microverse in order to help me practice what I learned in the first module. 
+
+## 🛠 Built With <a name="built-with"></a>
+Vs Code using HTML, CSS and JavaScript.
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- [Feature_1]: Setup a professional Github repository
+- [Feature_2]: Setup microverse linters
+- [Feature_3]:Use images and backgrounds to enhance the look of the website.
+- [Feature_4]: Use JavaScript to manipulate DOM elements
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] As per the previous reviewer's suggestion kindly add a section named `Live Demo` and add a link in this section. As for a professional README file a `Live Link` is must required. Kindly fix this issue.
- ![image](https://github.com/mariegrace31/capstone_one/assets/60083437/6c8f5653-2587-4586-bc80-da0ce1a49b69)